### ISSUE TITLE
Sensor fusion type error

### ISSUE THREAD

**Issue by bgottula** [seeing-things/track#214]
```
Traceback (most recent call last):
  File "/home/rgottula/dev/venv/bin/track", line 11, in <module>
    load_entry_point('track', 'console_scripts', 'track')()
  File "/home/rgottula/dev/track/track/__main__.py", line 134, in main
    tracker.run()
  File "/home/rgottula/dev/track/track/control.py", line 499, in run
    self.target.process_sensor_data()
  File "/home/rgottula/dev/track/track/targets.py", line 726, in process_sensor_data
    if np.abs(self.blind_target_bias) > self.bias_mag_limit.deg:
  File "/home/rgottula/dev/venv/lib/python3.6/site-packages/astropy/units/quantity.py", line 831, in __getattr__
    attr))
AttributeError: 'Quantity' object has no 'deg' member
```

**Comment by bgottula** [comment]
Line 620 of targets.py

`bias_mag_limit: Angle = 1.0*u.deg,`

should be

`bias_mag_limit: Angle = Angle(1.0*u.deg),`

**Comment by bgottula** [comment]
Fixed in 4fa475c0
